aria-practices.html

```diff
@@ -6421,6 +6421,305 @@ <h3>Indicating sort order with <code>aria-sort</code></h3>
+            <td>0</td>
+            <td>100</td>
+            <td>No</td>
+          </tr>
+          <tr>
+            <th><code>slider</code></th>
+            <td>0</td>
+            <td>100</td>
+            <td>No</td>
+          </tr>
+          <tr>
+            <th><code>spinbutton</code></th>
+            <td>no default</td>
+            <td>no default</td>
+            <td>No</td>
+          </tr>
+        </tbody>
+      </table>
+    </section>
+
+    <section id="range_related_properties_using_aria-valuenow">
+      <h2>Using <code>aria-valuenow</code></h2>
+      <p>
+        The <code>aria-valuenow</code> property is used to inform assistive technologies of the current value of a range widget.
+        If a range widget has a known value, set <code>aria-valuenow</code> to that value.
+        Otherwise, omit the <code>aria-valuenow</code> property.
+        If <code>aria-valuemin</code> and <code>aria-valuemax</code> are specified, ensure the value of <code>aria-valuenow</code> falls within the defined range.
+      </p>
+      <p>
+        The <code>aria-valuenow</code> property is required for the <code>scrollbar</code>, <code>separator</code> (if the element is focusable), and <code>slider</code> roles.
+      </p>
+    </section>
+
+    <section id="range_related_properties_using_aria-valuetext">
+      <h2>Using <code>aria-valuetext</code></h2>
+
+      <p>
+        When the element's values are contained within a range but those values are not numeric (such as "small", "medium" and "large"),
+        or they are numeric but there is more information than just the number,
+        <code>aria-valuetext</code> is used to surface the text value to assistive technologies.
+        Only use <code>aria-valuetext</code> when <code>aria-valuenow</code> does not have meaning for the user because using <code>aria-valuetext</code> will prevent assistive technologies from communicating <code>aria-valuenow</code>.
+      </p>
+
+      <div class="example">
+        <p>
+          For example, for a battery indicator, it would be useful to know how many minutes are remaining, in addition to how many percent charge is remaining.
+        </p>
+
+        <pre><code>&lt;p>
+  &lt;span id="battery-label">Battery&lt;/span>
+  &lt;span role="meter" aria-labelledby="battery-label"
+        aria-valuemin="0" aria-valuenow="5" aria-valuemax="100"
+        aria-valuetext="5%, 18 minutes remaining.">&lt;/span>
+&lt;/p></code></pre>
+      </div>
+    </section>
+
+    <section id="range_related_properties_meter_role">
+      <h3>Range properties with meter</h3>
+
+      <p>
+        A meter is a graphical display of a numeric value within a defined range. Therefore, the <code>aria-valuemin</code>, <code>aria-valuenow</code> and <code>aria-valuemax</code> properties are required for the <code>meter</code> role although there are no default values for <code>aria-valuemin</code> and <code>aria-valuemax</code>.
```

Comment:
There's no default value for aria-valuenow, because it's required.

So I think we could change this to something like:
```suggestion
        A meter is a graphical display of a numeric value within a defined range. The <code>aria-valuemin</code>, <code>aria-valuenow</code> and <code>aria-valuemax</code> properties are required for the <code>meter</code> role, and they don't have default values.
```